website/docusaurus.config.js

```diff
@@ -781,6 +781,22 @@ module.exports = {
                         ],
                         to: '/how-to/how-to-environment-import-export',
                     },
+                    {
+                        from: [
+                            '/topics/feature-flags/runtime-control',
+                            '/topics/feature-flags/never-expose-pii',
+                            '/topics/feature-flags/evaluate-flags-close-to-user',
+                            '/topics/feature-flags/scale-horizontally',
+                            '/topics/feature-flags/limit-payloads',
+                            '/topics/feature-flags/availability-over-consistency',
+                            '/topics/feature-flags/short-lived-feature-flags',
+                            '/topics/feature-flags/unique-names',
+                            '/topics/feature-flags/democratize-feature-flag-access',
+                            '/topics/feature-flags/prioritize-ux',
+                            '/topics/feature-flags/enable-traceability',
+                        ],
+                        to: '/topics/feature-flags/feature-flag-best-practices',
```

Comment:
Keep in mind that these redirects are purely client side, which generally is bad for SEO. We have established server side redirects here: https://github.com/Unleash/unleash/blob/main/website/vercel.json. 

(On that note we are getting quite a few redirects in now, we need to review them over time as they add complexity and there might be limits at Vercel that we are not aware of, its not uncommon to have an upper limits on these rules to keep performance under control.)

Reply:
I'll bring this up with @riboflavin so we can run an audit on our existing redirects to see if they get much traffic. Then we can start to trim down the list